### DISCUSSION TITLE
Gmail account setup issues

### DISCUSSION THREAD

**Discussion by skgrizz** [elementary/mail#801] [Q&A]
I've tried setting up my gmail account to receive email and I've followed the directions based on what google requires for an imap connection. I keep getting an error that I need to be online but I am connected to the network so I'm not sure what the issue could be. Any ideas?

**Comment by marbetschar** [comment]
Hi @skgrizz,

I assume you are receiving this error in `System Settings > Online Accounts`? Did you enable access for ["Less secure apps"](https://myaccount.google.com/lesssecureapps)?

If so, you are maybe experiencing a side effect of [this GMAIL Authentication Failure](https://github.com/elementary/mail/issues/789). Unfortunately I did not have time to dig into this yet, but I'm hoping to do so soon.

Meanwhile, you can try to install Evolution (as `*.deb` using `apt`) and setup the account there. Sorry to have no better news atm.

**Reply by RoboGR00t** [reply]
Gmail has disable the "Less secure app" feature and the validation must be done through googles SSO , that elementary mail client does not support it yet !

**Comment by MarcoPTTeixeira** [comment]
Hi all, 

I followed the instructions on the link below and it works fine. 

https://dev.to/k0ple/set-up-online-accounts-on-elementary-os-7cn
